project/AkkaDisciplinePlugin.scala

```diff
@@ -0,0 +1,191 @@
+package akka
+
+import sbt._, Keys._
+
+object CommonOptions {
```

Comment:
For any modules not wanting discipline, though I hope eventually to apply to all, these common settings can just be used. It is almost exactly what is in master, with minimal change.